.github/workflows/advisory-cron.yaml

```diff
@@ -0,0 +1,18 @@
+name: ci
+on:
+  schedule:
+    - cron: '0 18 * * *'
+jobs:
+  cargo-deny:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

Comment:
This does more than just advisories, which is fine but I think it should fail if there are any issues so we can be alerted more easily.

Reply:
Good shout, removed the `continue-on-error`.